CRM/Core/Resources.php

```diff
@@ -871,26 +872,23 @@ public static function renderMenubarStylesheet(GenericHookEvent $e) {
       "css/menubar-$cms.css",
     ];
     foreach ($items as $item) {
-      $e->content .= file_get_contents(self::singleton()->getPath('civicrm', $item));
-    }
-    $color = $e->params['color'];
-    if (!CRM_Utils_Rule::color($color)) {
-      $color = Civi::settings()->getDefault('menubar_color');
+      $content .= file_get_contents(self::singleton()->getPath('civicrm', $item));
     }
+    $params = $e->params;
+    // "color" is deprecated in favor of the more specific "menubarColor"
+    $menubarColor = $params['color'] ?? $params['menubarColor'];
```

Comment:
If color (sic) is deprecated why is it the 'first choice' here?

Reply:
Because if someone has implemented this hook to alter the "color" param, then we should continue to honor that. Otherwise we use the new param.

Reply:
ok